plugins/templates/list.js

```diff
@@ -58,18 +67,23 @@ module.exports = () => ({
                     field: fieldsToSearch,
                     // Do a fuzzy search for template name
                     // See https://www.w3schools.com/sql/sql_like.asp for syntax
-                    keyword: `%${request.query.search}%`
+                    keyword: `%${search}%`
                 };
             }
 
-            if (request.query.page || request.query.count) {
-                config.paginate = {
-                    page: request.query.page,
-                    count: request.query.count
-                };
+            // check if the call wants compact data
+            if (compact === '1') {
```

Comment:
why `1` vs `true`/`false`?

Reply:
I don't have a strong preference either way
I think we usually stick to `true`/`false`

Reply:
i can change

Reply:
kk, will change